src/rendering/vcTileRenderer.cpp

```diff
@@ -1440,8 +1440,9 @@ void vcTileRenderer_Render(vcTileRenderer *pTileRenderer, const udDouble4x4 &vie
     vcTileRenderer_RecursiveBuildRenderList(pTileRenderer, pRootNode, layer, nullptr, nullptr);
   }
 
-  // variable skirt length to handle logZ when close (tile logZ is done in vertex shader)
-  float tileSkirtLength = 8000.0f + (float)pTileRenderer->cameraDistanceToSurface;
+  // Variable skirt length to handle logZ when close (tile logZ is done in vertex shader)
+  const float EarthRadius = 6378137.0f; // TODO: Read this in
```

Comment:
This TODO is important enough to do now. The next sprint has non-earth planets that will break this immediately.